test/api_status_app_version_test.dart

```diff
@@ -1,9 +1,9 @@
-import 'package:openapi/api.dart';
+import 'package:tba_api_client/api.dart';
 import 'package:test/test.dart';
 
 // tests for APIStatusAppVersion
 void main() {
-    var instance = new APIStatusAppVersion();
```

Comment:
Is dart whitespace-sensitive like python, or were you just tidying up a bit with these?

Reply:
When I dartfmted the project, it made those changes

Reply:
ah okay, so just a formatter.  Makes sense.